packages/jaeger-ui/src/components/common/GraphSearch.js

```diff
@@ -0,0 +1,109 @@
+// @flow
```

Comment:
Seems like it might be nice to standardize naming where "search" is the search form and jaeger-query related and "find" is local to the UI. What do you think of renaming this to `GraphFind`?